wikimedia/wikipedia-ios
# Explore feed notifications onboarding card

**Phabricator:**  https://phabricator.wikimedia.org/T288693

### Notes
* The explore feed notification announcement card was updated to b used to alert users about the new notifications center feature. This PR updates the UI and the actions available from this card.

### Test Steps
1. Log in to your account
2. (It's necessary to wait for the data part to be done, a PR is coming up soon with these changes)
3. Check if the three-dot button actions work properly on the card (as it works on every other explore feed card)
4. Check if the "Turn on notifications" button takes you to the Notifications Center settings and  if the "dismiss" button removes the card from the feed

### Screenshots/Videos
